src/test/clojure/xtdb/indexer_test.clj

```diff
@@ -100,10 +101,13 @@
 
           (t/is (= {:latest-completed-tx last-tx-key}
                    (-> (meta/latest-block-metadata mm)
-                       (dissoc :tables :next-block-idx))))
+                       (dissoc :tables :block-idx))))
 
           (tj/check-json (.toPath (io/as-file (io/resource "xtdb/indexer-test/can-build-block-as-arrow-ipc-file-format")))
-                         (.resolve node-dir "objects")))))))
+                         (.resolve node-dir "objects"))
+
+          (cpb/check-pbuf  (.toPath (io/as-file (io/resource "xtdb/indexer-test/can-build-block-as-arrow-ipc-file-format")))
+                           (.resolve node-dir "objects")))))))
```

Comment:
I wasn't sure where to add the `check-pbuf` checks and where not. Most tests using `check-json` seem to be testing something different.

Reply:
for indexer test: I'd add them in most places here tbh - this is the main place that we really test what block-level files get written to disk.